modules/oe_webtools_analytics/modules/oe_webtools_analytics_rules/src/WebtoolsAnalyticsRuleListBuilder.php

```diff
@@ -17,6 +17,7 @@ class WebtoolsAnalyticsRuleListBuilder extends ConfigEntityListBuilder {
    */
   public function buildHeader(): array {
     $header['section'] = $this->t('Section');
+    $header['multilingual'] = $this->t('Multilingual');
```

Comment:
This should be renamed and probably made optional if the path module is not enabled.

Reply:
Fixed